content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -186,8 +185,8 @@ Resource associated with Activity:
     service.instance.id: 45aacfb0-e117-40cb-9d4d-9bcca661f6dd
 ```
 
-This output has both the span created to track work in the route, and an
-automatically-created span that tracks the inbound ASP.NET Core request itself.
+This output has both the span created to track work in the route,
+and a span that tracks the inbound ASP.NET Core request itself.
```

Comment:
The span is automatically created on behalf of the user, though. I'm not sure why this part was removed.

Reply:
instrumentation library created it

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry.io/pull/1565/commits/739e1298b8d9df6145d4dc33992f1b2f335c0e24